### DISCUSSION TITLE
List of graphics and songs made for this protest in website

### DISCUSSION THREAD

**Discussion by saeidkazekage** [WeAreMahsaAmini/Core#408] [Ideas]
Art is intertwined with revolution and always helps and facilitates.
The first and fastest helpers are graphic artists, poets and musicians.
Let's gather together the graphics and music created in these protests so that the impact of the situation becomes clearer and the value of accompanying graphic artists and musicians becomes more and more transparent, even if they are nameless.

**Comment by karimifar** [comment]
There is a google drive folder that's been circulating around social media:
https://drive.google.com/drive/u/2/folders/1DgRBz8EFMHHuo4SXqeZaeOsgCKTvewIo
